src/PROPOSAL/PROPOSAL/Secondaries.h

```diff
@@ -101,8 +101,10 @@ class Secondaries {
     std::vector<ContinuousLoss> GetContinuousLosses(const Geometry&) const;
 
 private:
-    ParticleState RePropagateDistance(const ParticleState&, double) const;
-    ParticleState RePropagateEnergy(const ParticleState&, double, double) const;
+    ParticleState RePropagateDistance(const ParticleState&, const Cartesian3D&,
```

Comment:
please add the parameter names, although this is not necessary, it improves the code readability; see the discussion [here](https://github.com/tudo-astroparticlephysics/PROPOSAL/pull/276#pullrequestreview-931656503)